qiskit_acqua/ising/maxcut.py

```diff
@@ -156,28 +156,24 @@ def sample_most_likely(n, state_vector):
     """Compute the most likely binary string from state vector.
 
     Args:
-        n (int): number of  qubits.
         state_vector (numpy.ndarray or dict): state vector or counts.
 
     Returns:
         numpy.ndarray: binary string as numpy.ndarray of ints.
     """
     if isinstance(state_vector, dict) or isinstance(state_vector, OrderedDict):
-        temp_vec = np.zeros(2**n)
-        total = 0
-        for i in range(2**n):
-            state = np.binary_repr(i, n)
-            count = state_vector.get(state, 0)
-            temp_vec[i] = count
-            total += count
-        state_vector = temp_vec / float(total)
-
-    k = np.argmax(np.abs(state_vector))
-    x = np.zeros(n)
-    for i in range(n):
-        x[i] = k % 2
-        k >>= 1
-    return x
+        # get the binary string with the largest count
+        binary_string = sorted(state_vector.items(), key=lambda kv: kv[1])[-1][0]
+        x = np.asarray([int(y) for y in list(binary_string)])
```

Comment:
@chunfuchen didn't you mean to also reverse the order of `x` before returning it? same goes for the other files as well

Reply:
My bad naming, I want to need the largest one only. 
Let me change the variable name from `binary_string` to `max_binary_string`.

Reply:
I only need the binary string with max count, and based on the sorted result, the one with the max count is allocated at the last one, and the binary string is stored at the first element according to the output format from qiskit.